### PR TITLE
chore(windows): breadcrumbs

### PR DESCRIPTION
**Pull request by mcdurdin**
Some of our Sentry reports are difficult to track down. Adding user interaction breadcrumbs to Keyman Configuration (to start with) may simplify this. I may add more breadcrumbs to other projects now that this is available.

Note: I removed a few unused dependencies as minor cleanup.

### CODE REVIEW

File: windows/src/ext/sentry/Sentry.Client.pas [hunk 1]
```diff
@@ -356,6 +358,18 @@ function TSentryClient.EventIDToString(AGuid: PByte): String;
...
 end;
 
+procedure TSentryClient.Breadcrumb(const BreadcrumbType, Message: string; const Category: string = ''; const Level: string = 'info');
+var
+  crumb: sentry_value_t;
+begin
+  crumb := sentry_value_new_breadcrumb(PAnsiChar(UTF8Encode(BreadcrumbType)), PAnsiChar(UTF8Encode(Message)));
+  if Category <> '' then
+    sentry_value_set_by_key(crumb, 'category', sentry_value_new_string(PAnsiChar(UTF8Encode(Category))));
+  if Level <> '' then
```
Comment (darcywong00) [inline windows/src/ext/sentry/Sentry.Client.pas:368]: Won't `Level` always be at least `'info'`?

**Review by darcywong00 — APPROVED** [review]
lgtm

File: windows/src/ext/sentry/Sentry.Client.pas (hunk 1 above)
Comment (mcdurdin) [reply windows/src/ext/sentry/Sentry.Client.pas:368]: Can pass in `''` as a parameter, in which case that overrides `'info'`, and just didn't want to pass empty string in that case.

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.20-alpha](https://keyman.com/downloads/releases/alpha/15.0.20)

**Pull request merged**